src/bsoncxx/string/view_or_value.hpp

```diff
@@ -41,6 +41,11 @@ class BSONCXX_API view_or_value : public bsoncxx::view_or_value<stdx::string_vie
     using bsoncxx::view_or_value<stdx::string_view, std::string>::view_or_value;
 
     ///
+    /// Default constructor, equivalent to using an empty string.
+    ///
+    BSONCXX_INLINE view_or_value() = default;
```

Comment:
I'm not sure why we didn't get this by forwarding bsoncxx::view_or_value's constructors?  It has a default constructor.